package.json

```diff
@@ -2,6 +2,7 @@
   "name": "inventory",
   "version": "0.0.1",
   "description": "keeping it simple.........",
+  "license": "MIT",
```

Comment:
This was just to make a npm warning go away... Do we care what license we release this with? I tend to like MIT.